module/os/linux/zfs/abd_os.c

```diff
@@ -1038,28 +749,14 @@ abd_release_ownership_of_buf(abd_t *abd)
 	/* Disable this flag since we no longer own the data buffer */
```

Comment:
I think that `abd_release_ownership_of_buf()` could be moved to the common `abd.c` if we made `abd_is_linear_page()` always return FALSE on FreeBSD, which would make sense since FreeBSD doesn't have this type of ABD.

Reply:
I went ahead a moved `abd_release_owership_of_buf()` to the common abd.c and placed `abd_is_linear_page()` in each of the OS specific ABD source files. In the case of FreeBSD I just always return B_FALSE. I also updated `abd_free_linear_page()` in the FreeBSD ABD source code to be `VERIFY(0)` as we should never get here.